x-pack/plugin/downsample/src/internalClusterTest/java/org/elasticsearch/xpack/downsample/ILMDownsampleDisruptionIT.java

```diff
@@ -172,6 +169,13 @@ public void testILMDownsampleRollingRestart() throws Exception {
         startDownsampleTaskViaIlm(sourceIndex, targetIndex);
         assertBusy(() -> assertTargetIndex(cluster, targetIndex, indexedDocs));
         ensureGreen(targetIndex);
+        // We assert that ILM successfully completed the phase
+        logger.info("Waiting for ILM to complete the phase for index [{}]", targetIndex);
+        awaitClusterState(clusterState -> {
```

Comment:
Ohh yes I'd completely forgotten we had an `awaitClusterState` to do this 👍

Reply:
Me too, credits to @nielsbauman